index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Week-1</title>
+  </head>
+  <body>
+    <h1>Hello My Name is Mohammed</h1>
+    <a href="https://www.youtube.com/watch?v=QNXvE1BZu8g&pp=ygUTcmlja3JvbGwgMTAgc2Vjb25kcw%3D%3D" alt="Link to see that app devs are the best">app devs are the best</a>
+    <p>Web devs are the best</p>
```

Comment:
Im sure you meant app
```suggestion
    <p>App devs are the best</p>
```

Reply:
Naaah web devs are the best